oci-unit-tests/helper/common_vars.sh

```diff
@@ -1,11 +1,14 @@
 # shellcheck shell=dash
 # shellcheck disable=SC2034
 
-readonly DOCKER_REGISTRY="${DOCKER_REGISTRY:-docker.io}"
-readonly DOCKER_NAMESPACE="${DOCKER_NAMESPACE:-ubuntu}"
-readonly DOCKER_PACKAGE="${DOCKER_PACKAGE:-$(basename "$0" | sed 's@\(.*\)_test\.sh@\1@')}"
-readonly DOCKER_TAG="${DOCKER_TAG:-edge}"
-readonly DOCKER_IMAGE="${DOCKER_IMAGE:-${DOCKER_REGISTRY}/${DOCKER_NAMESPACE}/${DOCKER_PACKAGE}:${DOCKER_TAG}}"
+[ -z "${DOCKER_REGISTRY}" ] && readonly DOCKER_REGISTRY="${DOCKER_REGISTRY:-docker.io}"
+[ -z "${DOCKER_NAMESPACE}" ] && readonly DOCKER_NAMESPACE="${DOCKER_NAMESPACE:-ubuntu}"
+[ -z "${DOCKER_PACKAGE}" ] && readonly DOCKER_PACKAGE="${DOCKER_PACKAGE:-$(basename "$0" | sed 's@\(.*\)_test\.sh@\1@')}"
+[ -z "${DOCKER_TAG}" ] && readonly DOCKER_TAG="${DOCKER_TAG:-edge}"
+[ -z "${DOCKER_IMAGE}" ] && readonly DOCKER_IMAGE="${DOCKER_IMAGE:-${DOCKER_REGISTRY}/${DOCKER_NAMESPACE}/${DOCKER_PACKAGE}:${DOCKER_TAG}}"
```

Comment:
I am not sure on why you had to add this "if $VAR empty => set VAR to itself if not-empty-or-null, or fallback to default value". Is this the ugly hack you mentioned in the PR description? I'm sure you have a good reason for it, but hopefully there's a nicer way...

Reply:
Yeah, this is the ugly hack.  It's needed because `test_helper.sh` will need to include `common_vars.sh`.  However, `common_vars.sh` is already being included by the `*_test.sh` scripts, which means that these variable will have been initialized already.  Since they're `readonly`, we get an error.

I have to say that I've basically decided to drop this PR and implement this in a single script, so we likely won't need this hack at all.